Casks/w/windows-app.rb

```diff
@@ -0,0 +1,41 @@
+cask "windows-app" do
+  version "11.0.1"
+  sha256 "8c53f6bc8f863c984967193c4f7572be80e0154cf90aa2db22303edcc3172faa"
+
+  url "https://officecdnmac.microsoft.com/pr/C1297A47-86C4-4C1F-97FA-950631F94777/MacAutoupdate/Windows_App_#{version}_installer.pkg",
+      verified: "officecdnmac.microsoft.com/pr/C1297A47-86C4-4C1F-97FA-950631F94777/MacAutoupdate/"
+  name "Windows App"
+  desc "Connect to Windows"
```

Comment:
This is a mere suggestion, but I was trying to come up with a more fitting description.

How about:

```suggestion
  desc "Connect to Windows desktops and applications"
```

or:

```suggestion
  desc "Connect to Windows devices and services"
```

Reply:
I took the descriptions from the [App Store tagline](https://aka.ms/macOSWindowsApp).